libkbfs/data_types.go

```diff
@@ -58,6 +58,7 @@ type TeamInfo struct {
 	TID          keybase1.TeamID
 	CryptKeys    map[KeyGen]kbfscrypto.TLFCryptKey
 	LatestKeyGen KeyGen
+	RootID       keybase1.TeamID // for subteams only
```

Comment:
Now that `RootID` is populated for root teams, I guess this comment should be changed accordingly?